web/src/pages/salesforce/index.js

```diff
@@ -0,0 +1,263 @@
+import React from "react";
+import Layout from "../../components/Layout/Layout";
+import { Container, Col, Row, Image } from "react-bootstrap";
+import Title from "../../components/UI/Title/Title";
+import Subtitle from "../../components/UI/Subtitle/Subtitle";
+import BrandButton from "../../components/UI/BrandButton/BrandButton";
+import "../../styles/main.scss"
+import styled from "styled-components";
+import Accordion from 'react-bootstrap/Accordion';
+import "./salesforce.scss";
+
+const SalesforcePage = ({ data }) => {
+    
+    return (
+        <Layout pageTitle="Salesforce | Awesome Inc">
+            
+            {/* Header section */}
+            <section id="header">
+                <Container fluid>
+                    <Row className="header-section">
+                        <Col className="header-text">
+                            <Title>SALESFORCE CAREER ACCELERATOR</Title>
+                            <BrandButton className="my-3">START YOUR APPLICATION</BrandButton>
+                            <p>Get started now in 30 seconds or less</p>
+                        </Col>
+                    </Row>
+                    <p className="paragraph">
```

Comment:
Let's use bootstrap `line-height` class here instead of `paragraph`. And the other utility classes to achieve the other class attributes